spec/std/zip/zip_spec.cr

```diff
@@ -0,0 +1,194 @@
+require "spec"
+require "zip"
+
+describe Zip do
+  it "writes and reads to memory" do
+    io = IO::Memory.new
+
+    Zip::Writer.open(io) do |zip|
+      zip.add "foo.txt", &.print("contents of foo")
+      zip.add "bar.txt", &.print("contents of bar")
+    end
+
+    io.rewind
+
+    Zip::Reader.open(io) do |zip|
+      entry = zip.next_entry.not_nil!
+      entry.file?.should be_true
+      entry.dir?.should be_false
+      entry.filename.should eq("foo.txt")
+      entry.compression_method.should eq(Zip::CompressionMethod::DEFLATED)
+      entry.crc32.should eq(0)
+      entry.compressed_size.should eq(0)
+      entry.uncompressed_size.should eq(0)
```

Comment:
Why is size zero here, if `foo.txt` has `"contents of foo"`? (keep in mind I'm far from being an expert in Zip though)